schema/cam/cam_schema_2-0-0.json

```diff
@@ -31,7 +31,7 @@
     "version": {
       "type": "string",
       "description": "json message format version",
-      "const": "1.99.99"
+      "const": "2.0.0"
```

Comment:
In fact, seen with Yann, it's a way to have only the right version number (2.0.0 in this case) in the very last commit including all the updates. If someone pull an "intermediate" commit, before the last one, he won't have the right version in the schema but a version with .99 indicating it's a developing version.

Reply:
Indeed, this avoids cases where two files, at two different revisions, have the same version:
1. if we add/delete/rename fields while still keeping the old version number, then we have multiple schemas with the old version, and it is not easy to find which is the correct one;
2. if we first change the version, then the same is true, but with the new version.

We can't really rely on the filenames, because filenames can change; we can only rely on the version in the schemas themselves. Thus, it is important to denote that a schema is not stable, and that it should not be used; this is the role of the _.99_ version, which clearly inidicates that.

When discussing this with @Hugues360, I subtly hinted that we should have documentation that explains our process at maintaining those schmas. Way too subtly, maybe... ;-) More generally, though, we should have a documentation that explains the versioning scheme for each component (java, python, rust, schemas).